tests/test-sessions/test-jdbc-sessions/src/test/java/org/eclipse/jetty/server/session/JdbcTestHelper.java

```diff
@@ -173,17 +173,17 @@ public static boolean checkSessionPersisted(SessionData data)
         ResultSet result = null;
         try (Connection con = DriverManager.getConnection(DEFAULT_CONNECTION_URL);)
         {
-            statement = con.prepareStatement("select * from " + TABLE +
-                " where " + ID_COL + " = ? and " + CONTEXT_COL +
-                " = ? and virtualHost = ?");
-            statement.setString(1, data.getId());
-            statement.setString(2, data.getContextPath());
-            statement.setString(3, data.getVhost());
+            statement = con.prepareStatement(
+                "select * from " + TABLE +
+                    " where " + ID_COL + " = ? and " + CONTEXT_COL +
+                    " = ? and virtualHost = ?" );
+            statement.setString(1, data.getId() );
```

Comment:
how are you getting away with the extra space before the final ); ??? Is checkstyle broken?

Reply:
ohh my bad I usually avoid only blank line. But agree I have to take care of spaces as well